lib/hooks/useAllCreatedPrizePoolsWithTokens.js

```diff
@@ -15,11 +15,13 @@ export const useAllCreatedPrizePoolsWithTokens = () => {
 
   return useQuery(
     [QUERY_KEYS.useAllCreatedPrizePoolsWithTokens, prizePools, chainId],
-    async () => getAllCreatedPrizePoolsWithTokens(provider, prizePools, chainId),
+    async () => await getAllCreatedPrizePoolsWithTokens(provider, prizePools, chainId),
     {
       enabled: readProviderIsLoaded && createdPrizePoolsIsFetched,
       refetchInterval: false,
-      refetchOnWindowFocus: false
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
```

Comment:
Let's put these 3 lines in a default react-query config constant and spread that into each object instance.